packages/@uppy/companion/src/server/provider/dropbox/index.js

```diff
@@ -20,6 +20,27 @@ function httpHeaderSafeJson (v) {
     })
 }
 
+function adaptData (res, email, buildURL) {
+  const items = adapter.getItemSubList(res).map((item) => ({
+    isFolder: adapter.isFolder(item),
+    icon: adapter.getItemIcon(item),
+    name: adapter.getItemName(item),
+    mimeType: adapter.getMimeType(item),
+    id: adapter.getItemId(item),
+    thumbnail: buildURL(adapter.getItemThumbnailUrl(item), true),
+    requestPath: adapter.getItemRequestPath(item),
+    modifiedDate: adapter.getItemModifiedDate(item),
+    size: adapter.getItemSize(item),
+  }))
+  items.sort((a, b) => a.name.localeCompare(b.name, 'en-US', { numeric: true }))
```

Comment:
This works too, but I'm sure the Dropbox API supports a query parameter to do it for us. Either way is fine I suppose.

Reply:
seems like it doesnt: https://www.dropboxforum.com/t5/Dropbox-API-Support-Feedback/list-orderby-api/td-p/337393